src/components/Animal.js

```diff
@@ -0,0 +1,11 @@
+const List = ({ list = [] }) => {
+  return (
+    <div className='p-40'>
+      {list.map((animal, index) => {
+        return (<ul key={index}>{animal}</ul>)
```

Comment:
you can write this in one line by removing the return keyword and curly braces which are not necessary